recipes-kernel/linux/files/android-drivers.cfg

```diff
@@ -1,8 +1,3 @@
-CONFIG_ANDROID=y
-
 #CONFIG_ANDROID_BINDERFS is not set
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_ANDROID_BINDER_DEVICES="binder"
-
-CONFIG_ASHMEM=y
```

Comment:
do we need to add `CONFIG_MEMFD_CREATE` after removing this ? or is it already enabled in defaults.

Reply:
MEMFD_CREATE is also enabled in 6.1.
It is supposed to be set to 'y' depending on TMPFS which is likely set as 'y' normally.

In fs/Kconfig:
```
config MEMFD_CREATE
    def_bool TMPFS || HUGETLBFS
```